packages/manager/apps/web/client/app/domain-operation/update/domain-operation-update.controller.js

```diff
@@ -192,115 +192,128 @@ angular.module('App').controller(
-                );
-            case 'accelerate':
-              return this.domainOperationService[
-                `accelerate${this.constructor.capitalizeFirstLetter(
-                  this.type,
-                )}Operation`
-              ](this.operation.id)
-                .then(() =>
-                  this.Alerter.success(
-                    this.$translate.instant(
-                      'domain_operations_accelerate_success',
-                    ),
-                    ALERTER_ID,
-                  ),
-                )
-                .catch((err) =>
-                  this.Alerter.alertFromSWS(
-                    this.$translate.instant(
-                      'domain_operations_accelerate_error',
-                    ),
-                    err,
-                    ALERTER_ID,
-                  ),
-                );
-            default:
-              this.Alerter.success(
-                this.$translate.instant('domain_tab_OPERATION_update_success'),
-                ALERTER_ID,
-              );
-              return null;
-          }
-        })
-        .catch((err) =>
+      // eslint-disable-next-line no-restricted-syntax
+      for (const promise of promises) {
+        try {
+          // eslint-disable-next-line no-await-in-loop
+          await promise();
+        } catch (error) {
           this.Alerter.alertFromSWS(
             this.$translate.instant('domain_tab_OPERATION_update_error'),
-            err,
+            error,
+            ALERTER_ID,
+          );
+          break;
+        }
+      }
+
+      switch (this.todoOperation) {
+        case 'relaunch':
+          await this.domainOperationService[
+            `relaunch${this.constructor.capitalizeFirstLetter(
+              this.type,
+            )}Operation`
+          ](this.operation.id)
+            .then(() =>
+              this.Alerter.success(
+                this.$translate.instant(
+                  'domain_tab_OPERATION_update_relaunch_success',
+                ),
+                ALERTER_ID,
+              ),
+            )
+            .catch((err) =>
+              this.Alerter.alertFromSWS(
+                this.$translate.instant('domain_operations_relaunch_error'),
+                err,
+                ALERTER_ID,
+              ),
+            );
+          break;
+        case 'cancel':
+          await this.domainOperationService[
+            `cancel${this.constructor.capitalizeFirstLetter(
+              this.type,
+            )}Operation`
+          ](this.operation.id)
+            .then(() =>
+              this.Alerter.success(
+                this.$translate.instant('domain_operations_cancel_success'),
+                ALERTER_ID,
+              ),
+            )
+            .catch((err) =>
+              this.Alerter.alertFromSWS(
+                this.$translate.instant('domain_operations_cancel_error'),
+                err,
+                ALERTER_ID,
+              ),
+            );
```

Comment:
There is some time Alerter.success and Alerter.alert
Maybe it's possible to avoid the duplication by creating dedicated function.